src/functionaltests/resources/ValidSubscription.json

```diff
@@ -2,6 +2,7 @@
   {
     "subscriptionName": "mySubscription",
     "userName" : "DEF",
```

Comment:
Not sure username should be here since it does not have BASIC_AUTH selected.

Reply:
Well, its presence or absence doesnt make any difference.... or?